internal/app/go-filecoin/paymentchannel/manager.go

```diff
@@ -59,26 +58,16 @@ type MsgSender interface {
 		params interface{}) (out cid.Cid, pubErrCh chan error, err error)
 }
 
-// MgrStateViewer is the subset of a StateViewer API that the Manager uses
-type MgrStateViewer interface {
-	StateView(root cid.Cid) PaychActorStateView
-}
-
-// PaychActorStateView is the subset of a StateView that the Manager uses
-type PaychActorStateView interface {
-	PaychActorParties(ctx context.Context, paychAddr address.Address) (from, to address.Address, err error)
-	MinerControlAddresses(ctx context.Context, addr address.Address) (owner, worker address.Address, err error)
-}
-
-// ChainReader is the subset of the ChainReadWriter API that the Manager uses
-type ChainReader interface {
-	GetTipSetStateRoot(context.Context, block.TipSetKey) (cid.Cid, error)
+// ActorStateViewer is an interface to StateViewer that the Manager uses
```

Comment:
This was refactored to a state viewer api because initializing the connector in node.go surfaced some conflicts between testing and the node code.